src/index_emails.py

```diff
@@ -213,7 +269,25 @@ def load_from_file():
                            help="Number of shards for ES index")
 
     tornado.options.define("index_bodies", type=bool, default=False,
-                           help="Will index all body content, stripped of HTML/CSS/JS etc. Adds fields: 'body' and 'body_size'")
+                           help="Will index all body content, stripped of HTML/CSS/JS etc. Adds fields: 'body', 'body_size' and 'body_filenames' for any multi-part attachments")
+
+    tornado.options.define("index_bodies_ignore_content_types", default="application,image",
```

Comment:
`index_bodies_ignore_content_types` is very long - think there's a shorter name that describes what's going on?

Reply:
Will consider it in next iteration, working on other features.